ros/src/twist_controller/twist_controller.py

```diff
@@ -42,14 +43,21 @@ def reset(self):
     def compute_acceleration(self, error):
         last_timestep = self.current_timestep
         self.current_timestep = int(round(time.time() * 1000))
-        
+
+        if (last_timestep == self.current_timestep):
+            rospy.loginfo(
+                'No timestep advance since last acceleration computation; returning previous acceleration')
+            return self.previous_acceleration
+
         # TODO - modify PID instance to modify error value to prevent sudden control jumps when re-engaging DBW
         # after a period of inactivity
         if last_timestep is None:
             # attempted to compute values for first time; do not accellerate
             return 0.
 
-        return self.velocity_pid_controller.step(error, self.current_timestep - last_timestep)
+        self.previous_acceleration = self.velocity_pid_controller.step(
+            error, self.current_timestep - last_timestep)
+        return self.previous_acceleration
```

Comment:
Could you use a temp variable to store all of the possible return values and then return that at the end?